fe/fe-core/src/main/java/com/starrocks/clone/Rebalancer.java

```diff
@@ -58,10 +57,10 @@ public Rebalancer(SystemInfoService infoService, TabletInvertedIndex invertedInd
 
     public List<TabletSchedCtx> selectAlternativeTablets() {
         List<TabletSchedCtx> alternativeTablets = Lists.newArrayList();
-        for (Map.Entry<String, ClusterLoadStatistic> entry : statisticMap.entrySet()) {
+        ClusterLoadStatistic localLoadStat = loadStatistic;
+        if (localLoadStat != null) {
```

Comment:
You can use `loadStatistic` directly.

Reply:
I'm afraid that loadStatisitc can be changed, so I got a reference.